server/plugin.go

```diff
@@ -256,6 +256,9 @@ func (p *Plugin) start(isRestart bool) {
 		if err != nil {
 			p.API.LogError("failed to start metrics job", "error", err)
 		}
+
+		// Run the job above right away so we immediately populate metrics.
+		go p.updateMetrics()
```

Comment:
We'd always get dips otherwise: 

![CleanShot 2024-07-25 at 17 30 45@2x](https://github.com/user-attachments/assets/f526e9ff-640c-406c-a6f1-206e83954b1f)